<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficFlow</class>
 <widget class="QWidget" name="TrafficFlow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetShow" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>刷新内容</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/myimages.qrc">
      <normaloff>:/new/imgsrc/re.png</normaloff>:/new/imgsrc/re.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>D显示一天的流量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMonth">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>M显示一月的流量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonYear">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y显示一年的流量</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>0</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>416</x>
      <y>0</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>0</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLater">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/myimages.qrc">
      <normaloff>:/new/imgsrc/later.png</normaloff>:/new/imgsrc/later.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFoward">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/myimages.qrc">
      <normaloff>:/new/imgsrc/foward.png</normaloff>:/new/imgsrc/foward.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images/myimages.qrc"/>
 </resources>
 <connections/>
</ui>
